src/routes/docs/products/functions/develop/+page.markdoc

```diff
@@ -1093,7 +1082,7 @@ You can access these logs through the following steps.
 If you need to pass constants or secrets to Appwrite Functions, you can use environment variables. 
 
 {% info title="Appwrite API keys" %}
-If your function is using an Appwrite SDK with an API key, this API key needs to be generated and passed in manually. API keys are not passed by default for security reasons.
+If your function uses an **Appwrite SDK** with a **dynamic API** key, go to the function's settings and specify what [**scopes**](/docs/advanced/platform/api-keys#scopes) the API key has access to.
 {% /info %}
```

Comment:
Let's get rid of this info block now since this to warn people about the API key, but now it's automatically passed.

Reply:
👆🏼

Reply:
If you take a look at my current code in the PR, the [info block](https://github.com/appwrite/website/pull/1045/files#diff-6f1ab87d531d6830f05abbdf86ca97b0ce4d413b80e9d393156f705837ff7d45L1095) has already been removed.